### PR TITLE
OSDOCS-12470:updating system requirements and Getting started concept…

### PR DESCRIPTION
**Pull request by aravipra**
Version(s):
4.17+

Issue:
[OSDOCS-12470](https://issues.redhat.com/browse/OSDOCS-12470)

Link to docs preview:
[System requirements for installing MicroShift](https://84722--ocpdocs-pr.netlify.app/microshift/latest/microshift_install_get_ready/microshift-install-get-ready.html)

QE review:
- [x] QE has approved this change.

SME review:
- [x] SME has approved this change.

### CODE REVIEW

**Review by DanielFroehlich — APPROVED** [review]
/LGTM

File: modules/microshift-install-system-requirements.adoc [hunk 1]
```diff
@@ -8,15 +8,22 @@
...
-* A compatible version of {op-system-base-full}.
+* A compatible version of {op-system-base-full}.For more information, see link:https://docs.redhat.com/en/documentation/red_hat_build_of_microshift/{ocp-version}/html/getting_ready_to_install_microshift/microshift-install-get-ready#get-ready-install-rhde-compatibility-table_microshift-install-get-ready[Compatibility table].
 * AArch64 or x86_64 system architecture.
 * 2 CPU cores.
-* 2 GB RAM for {microshift-short} or 3 GB RAM, required by {op-system-base} for networked-based HTTPs or FTP installations.
+* 2 GB RAM. Installing from the network (UEFI HTTPs or PXE boot) requires 3 GB RAM for {op-system-base}.
 * 10 GB of storage.
 * You have an active {microshift-short} subscription on your Red Hat account. If you do not have a subscription, contact your sales representative for more information.
-* You have a Logical Volume Manager (LVM) Volume Group (VG) with sufficient capacity for the Persistent Volumes (PVs) of your workload.
+* When your workload requires Persistent Volumes (PVs), you have a Logical Volume Manager (LVM) Volume Group (VG) with sufficient free capacity for the workloads.
```
Comment (mburke5678) [inline modules/microshift-install-system-requirements.adoc:17]: _When_ sounds like a reference to time. I don't think that is the case. 

```suggestion
* If your workload requires Persistent Volumes (PVs), you have a Logical Volume Manager (LVM) Volume Group (VG) with sufficient free capacity for the workloads.
```

File: modules/microshift-install-system-requirements.adoc [hunk 2]
```diff
@@ -8,15 +8,22 @@
...
+
+[NOTE]
+====
+These requirements are the minimum system requirements for {microshift-short} and {op-system-base-full}. Add the system requirements for the workload you plan to run.
+For example, if an IoT gateway solution requires 4 GB of RAM, your system needs to have at least 2GB for {op-system-base-full} and {microshift-short}, plus 4GB for the workloads. 6GB of RAM is required in total.
```
Comment (mburke5678) [inline modules/microshift-install-system-requirements.adoc:22]: It took me a minute or 5 to figure it out, as I don't know what an IoT gateway solution is. But the workloads here are for the IoT gateway solution? Or is it 4 for the gateway, 2 for the {op-system-base-full} and {microshift-short}, **_and_** 4 more for workloads??  
```suggestion
For example, if an IoT gateway solution requires 4 GB of RAM, your system needs to have at least 2 GB for {op-system-base-full} and {microshift-short}, plus 4 GB for the workloads. 6 GB of RAM is required in total.
```

**Comment by mburke5678** [comment]
@aravipra A few comments. Otherwise LGTM

File: modules/microshift-install-system-requirements.adoc (hunk 1 above)
Comment (aravipra) [reply modules/microshift-install-system-requirements.adoc:17]: done

File: modules/microshift-install-system-requirements.adoc (hunk 2 above)
Comment (aravipra) [reply modules/microshift-install-system-requirements.adoc:22]: done

**Pull request merged**